app/views/helpers/lists.php

```diff
@@ -470,7 +470,7 @@ public function displayAddSentenceForm($listId)
                 'class' => 'submit'
             )
         );
-        echo "<div class='loader-container'></div>";
+        echo "<md-progress-circular id='list_add_loader' md-diameter='11px' style='display: none;'></md-progress-circular>";
```

Comment:
`md-diameter` can be increased to 16px. 11px is a bit too small.
Also note that you don't need to specify "px". `md-diameter="16"` works too.